README.rst

```diff
@@ -86,6 +86,11 @@ You must define a palette and each row can display 4 colors::
     
     $ omero config set omero.web.iviewer.show_palette_only true
 
+When working with other images (coregistering MRIs for example), it is necessary to be able to mirror an image.
+There is now experimental support for runtime image mirroring. To enable mirroring set enable_mirror to true.
+
+    $ omero config set omero.web.iviewer.enable__mirror true
```

Comment:
You have a double underscore in `enable__mirror` which I think is why my config didn't work before.